packages/cli/src/models/dependency/Dependency.ts

```diff
@@ -1,58 +1,67 @@
 import _ from 'lodash';
-import { FileSystem as fs, PackageProject, Contracts, getSolidityLibNames, Logger } from 'zos-lib'
+import { FileSystem as fs, PackageProject, Contracts, getSolidityLibNames, Logger } from 'zos-lib';
 import semver from 'semver';
-import npm from 'npm-programmatic'
+import npm from 'npm-programmatic';
 
 import ZosPackageFile from '../files/ZosPackageFile';
 import ZosNetworkFile from '../files/ZosNetworkFile';
 
 const log = new Logger('Dependency');
 
 export default class Dependency {
-  static fromNameWithVersion(nameAndVersion) {
-    const [name, version] = nameAndVersion.split('@')
-    return new this(name, version)
+
+  public name: string;
+  public version: string;
+  public nameAndVersion: string;
+  public requirement: string | semver.Range;
+
+  private _networkFiles: { [network: string]: ZosNetworkFile };
+  private _packageFile: ZosPackageFile;
+
+  public static fromNameWithVersion(nameAndVersion: string): Dependency {
+    const [name, version] = nameAndVersion.split('@');
+    return new this(name, version);
   }
 
-  static satisfiesVersion(version, requirement) {
+  public static satisfiesVersion(version: string | semver.SemVer, requirement: string | semver.Range): boolean {
```

Comment:
Can we import `SemVer` and `Range` in `L3` like this?
```ts
import semver, { SemVer, Range } from 'semver';
```

Reply:
Yes, we can do that, although I think typing 3rd party stuff with namespaces is clearer. Will add a note to my list of things to do after the merge. I'll ping you for discussing this.